scene/animation/animation_tree.cpp

```diff
@@ -281,7 +281,7 @@ double AnimationNode::_blend_node(const StringName &p_subpath, const Vector<Stri
 	if (r_max) {
 		*r_max = 0;
 		for (int i = 0; i < blend_count; i++) {
-			*r_max = MAX(*r_max, blendw[i]);
+			*r_max = MAX(*r_max, Math::abs(blendw[i]));
```

Comment:
Fixed a problem in which graph lines were not colored when blend values were negative, even though blending occurred.

![image](https://user-images.githubusercontent.com/61938263/235362469-0829fa9a-3dea-4712-908a-2ce1943b61ab.png)

This r_max only affects the activity value and not the actual blend value.